README.md

```diff
@@ -67,6 +67,56 @@ const webhook = new SeamWebhook('webhook-secret')
 const data = webhook.verify(payload, headers)
 ```
 
+#### Receiving Webhooks
+
+First, create a webhook using the Seam API or Seam Console
+and obtain a Seam webhook secret.
+
+_This example is for [Express], see the [Svix docs for more examples in specific frameworks](https://docs.svix.com/receiving/verifying-payloads/how)._
+
+```js
+import { SeamWebhook } from 'seam'
+import express from 'express'
+import bodyParser from 'body-parser'
+
+import { storeEvent } from './store-event.js'
+
+const app = express()
+
+const webhook = new SeamWebhook(process.env.SEAM_WEBHOOK_SECRET)
+
+app.post(
+  '/webhook',
+  bodyParser.raw({ type: 'application/json' }),
+  (req, res) => {
+    let data
+    try {
+      data = webhook.verify(payload, headers)
+    } catch {
+      return res.status(400).send()
+    }
+
+    storeEvent(data, (err) => {
+      if (err != null) {
+        return res.status(500).send()
+      }
+      res.status(204).send()
+    })
+  },
+)
+```
+
+[Express]: https://expressjs.com/
+
+#### Using HTTP client
```

Comment:
Looks like we need to grab the rest of the section. It should inject down to the end 
![image](https://github.com/seamapi/javascript-next/assets/721372/3de46220-992d-4b03-b586-c839fa6f5c34)

Reply:
This should do it


```ts
  const regex = /#* Usage\s*([\s\S]*?(?=\n## \w))/